composeApp/src/commonMain/kotlin/org/ooni/probe/data/models/InstalledTestDescriptorModel.kt

```diff
@@ -56,6 +80,8 @@ data class InstalledTestDescriptorModel(
         val revision: Long,
     )
 
+    val isDefaultTestDescriptor get() = OoniTest.isValidId(id.value.toLongOrNull() ?: -1L)
```

Comment:
Not sure if we should keep with the default concept or not. It's confusing with the NewsMediaScan descriptors, which are default, but not OONI tests. Maybe split this between `isOoniDescriptor` and `isUninstallable`.